### PR TITLE
Upgrade Flutter Version

### PR DESCRIPTION
**Pull request by PormpimonPang**
Our project use channel stable:
- Upgrade Flutter 2.2.3 
- Upgrade Dart version: 2.13.4
- Migrate FlatButton to TextButton 
- Migrate RaisedButton to  ElevatedButton

### CODE REVIEW

File: lib/pages/chat_page.dart [hunk 1]
```diff
@@ -175,55 +174,55 @@ class _ChatPageState extends State<ChatPage> {
...
+    //     text: text,
+    //     name: "Patient",
+    //     type: true,
+    //   );
+    //   setState(() {
+    //     _messageModel.insertMessage(0, message);
+    //   });
+    //   response(text);
+    //   _textController.clear();
+    // }
```
Comment (Thaweesak-sai) [inline lib/pages/chat_page.dart:226]: These lines will be fixed by the next pr right ?

File: lib/widget/chatbot/chat_training.dart [hunk 2]
```diff
@@ -46,20 +46,36 @@ class _ChatTrainingState extends State<ChatTraining> {
...
+            //             bottomRight: Radius.circular(15.0))),
+            //     color: Color(0xFFF1B43F),
+            //     onPressed: () {
+            //       Navigator.push(
+            //           context,
+            //           MaterialPageRoute(
+            //               builder: (context) =>
+            //                   DigestiveAdviceDay2(navigate: navigate)));
+            //     },
+            //     child: chatTrainingButton),
```
Comment (Thaweesak-sai) [inline lib/widget/chatbot/chat_training.dart:78]: If migrated. remove it

File: lib/widget/chatbot/chat_training.dart [hunk 3]
```diff
@@ -93,20 +109,36 @@ class _ChatTrainingState extends State<ChatTraining> {
...
+            //             bottomRight: Radius.circular(15.0))),
+            //     color: Color(0xFFF1B43F),
+            //     onPressed: () {
+            //       Navigator.push(
+            //           context,
+            //           MaterialPageRoute(
+            //               builder: (context) =>
+            //                   PulmonaryAdviceDay2(navigate: navigate)));
+            //     },
+            //     child: chatTrainingButton),
```
Comment (Thaweesak-sai) [inline lib/widget/chatbot/chat_training.dart:141]: This one too

File: lib/widget/chatbot/chat_training.dart [hunk 4]
```diff
@@ -137,20 +169,36 @@ class _ChatTrainingState extends State<ChatTraining> {
...
+            //             bottomRight: Radius.circular(15.0))),
+            //     color: Color(0xFFF1B43F),
+            //     onPressed: () {
+            //       Navigator.push(
+            //           context,
+            //           MaterialPageRoute(
+            //               builder: (context) =>
+            //                   DrainSecretionAdviceDay2(navigate: navigate)));
+            //     },
+            //     child: chatTrainingButton),
```
Comment (Thaweesak-sai) [inline lib/widget/chatbot/chat_training.dart:201]: this one as well

File: lib/widget/rich_menu.dart [hunk 5]
```diff
@@ -50,18 +50,11 @@ class _RichMenuState extends State<RichMenu> {
...
-                },
+              TextButton(
+                style: TextButton.styleFrom(backgroundColor: Color(0xFFFCECBC)),
                 child: Container(
                   height: 100,
                   width: MediaQuery.of(context).size.width,
-                  decoration: BoxDecoration(
-                    borderRadius: BorderRadius.circular(0),
-                    color: Color(0xFFFCECBC),
-                  ),
```
Comment (Thaweesak-sai) [inline lib/widget/rich_menu.dart:64]: This could be design change? or the migration does not complete here?

**Comment by Thaweesak-sai** [comment]
@PormpimonPang  Leave some comments.

File: lib/pages/chat_page.dart (hunk 1 above)
Comment (PormpimonPang) [reply lib/pages/chat_page.dart:226]: Yes

File: lib/widget/rich_menu.dart (hunk 5 above)
Comment (PormpimonPang) [reply lib/widget/rich_menu.dart:64]: Screen Shot 2564-07-19 at 19 54 50

The UI result remains the same, but the richmanu is migrated with textbutton, so the code has changed a bit.

File: lib/widget/chatbot/chat_training.dart (hunk 2 above)
Comment (PormpimonPang) [reply lib/widget/chatbot/chat_training.dart]: c4aa8de Fixed ka

File: lib/widget/chatbot/chat_training.dart (hunk 3 above)
Comment (PormpimonPang) [reply lib/widget/chatbot/chat_training.dart]: c4aa8de Fixed ka

File: lib/widget/chatbot/chat_training.dart (hunk 4 above)
Comment (PormpimonPang) [reply lib/widget/chatbot/chat_training.dart]: c4aa8de Fixed ka

**Review by Thaweesak-sai — APPROVED** [review]

**Pull request merged**